frontend/packages/shipwright-plugin/src/actions/useBuildActions.ts

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { getCommonResourceActions } from '@console/app/src/actions/creators/common-factory';
+import { Action } from '@console/dynamic-plugin-sdk/src/extensions/actions';
+import { errorModal } from '@console/internal/components/modals';
+import { resourceObjPath } from '@console/internal/components/utils';
+import { referenceFor } from '@console/internal/module/k8s';
+import { useK8sModel } from '@console/shared/src/hooks/useK8sModel';
+import { startBuild } from '../api';
+import { Build } from '../types';
+
+export default function useBuildActions(build: Build) {
```

Comment:
At @jerolimov this is also fine but to maintain code consistency you can use change it to an arrow function.
```suggestion
const useBuildActions = (build: Build) => {
```